nextcloud_odoo_sync/models/nc_calendar.py

```diff
@@ -1,19 +1,22 @@
-# -*- coding: utf-8 -*-
 # Copyright (c) 2023 iScale Solutions Inc.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
 from odoo import models, fields, api
 
 
-class NcsyncUser(models.Model):
-    _name = 'nc.calendar'
+class NcCalendar(models.Model):
+    _name = "nc.calendar"
+    _description = "Nextcloud Calendar"
 
-    name = fields.Char(string='Calendar')
-    user_id = fields.Many2one('res.users', string='User', ondelete='cascade')
-    calendar_url = fields.Text(string='Calendar URL')
+    name = fields.Char(string="Calendar", required=True)
+    user_id = fields.Many2one(
+        "res.users", string="User", ondelete="cascade", required=True
+    )
+    calendar_url = fields.Text(string="Calendar URL")
 
     @api.model
     def create(self, vals):
-        if not self._context.get('sync', False):
-            vals['user_id'] = self.env.user.id
-        return super(NcsyncUser, self).create(vals)
+        if not self._context.get("sync", False):
```

Comment:
```suggestion
        if not self._context.get("sync", False) and "user_id" not in vals:
```
Should be done in one line